odoo/addons/base/models/res_lang.py

```diff
@@ -249,6 +249,25 @@ def _lang_code_to_urlcode(self, code):
                 return urlc
         return self._lang_get(code).url_code
 
+    @api.model
+    def _get_enabled_lang_code(self):
+        """ Get the list of code enabled (aka installed and default) """
+        langs = self.get_enabled()
+        return [lang[0] for lang in langs]
+
+    @api.model
+    @tools.ormcache()
```

Comment:
Don't we have to somehow provide an invalidation of the cache ? like code, iso_code and active ?

Reply:
Other caches methods generally give triggers-like stuff. I don't know the internals of the ormcache, just pointing it out. Maybe @mart-e could give its word ?

Reply:
The other methods of res.lang have triggers indeed, something like `@tools.ormcache('iso_code')` should do it. But then we do not change very frequently the res.lang records, so not a big deal I guess...